root_locals.tf

```diff
@@ -100,6 +100,7 @@ locals {
   runtime_java_11    = "java11"
 
   upload_files_cloudfront_dirty_bucket_name = "${var.project}-upload-files-cloudfront-dirty-${local.environment}"
+  draft_metadata_bucket_name                = "${var.project}-draft-metadata-${local.environment}"
```

Comment:
Is there a reason to define the bucket name here? Its only used within the `root_draft_metadata.tf` I would define it as a `local` within the `rootf_draft_metadata.tf` unless its needed else where in the code.

Reply:
Yeah, we can define it within the `rootf_draft_metadata.tf`